### PR TITLE
platform/mikro-e/dev: add motion click driver and update relay click driver

### PR DESCRIPTION
**Pull request by mayank-sirotiya-imgtec**
This PR
1. Adds motion click driver.
2. Adds motions click test application.
3. Restructures relay click driver files.
4. Updates relay click test application as per restructured relay click driver files.

@IMGCreator/creators 

Signed-off-by: Mayank Sirotiya Mayank.Sirotiya@imgtec.com
Signed-off-by: Onkar Kshirsagar onkarkshirsagar3@gmail.com

### CODE REVIEW

File: platform/mikro-e/dev/motion-click/Makefile.motion-click [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+ifdef MOTION_CLICK
```
Comment (manohar-narkhede-imgtec) [inline platform/mikro-e/dev/motion-click/Makefile.motion-click]: This flag is not required in this Makefile

File: platform/mikro-e/dev/motion-click/motion-click.h [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  do {                                                                  \
+    GPIO_CONFIGURE_AS_INPUT(D, 0);                                      \
+    CNCONDSET = _CNCOND_ON_MASK;                                        \
+    IEC1SET = _IEC1_CNDIE_MASK;                                         \
+    IFS1CLR = _IFS1_CNDIF_MASK;                                         \
+    IPC8SET = (6 << _IPC8_CNIP_POSITION) | (0 << _IPC8_CNIS_POSITION);  \
+    (void)PORTD;                                                        \
+  } while(0)
+
+/* To enable Motion sensor interupt */
```
Comment (manohar-narkhede-imgtec) [inline platform/mikro-e/dev/motion-click/motion-click.h]: spelling of interrupt..

**Comment by manohar-narkhede-imgtec** [comment]
Approved and Verified.

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

**Pull request merged**